src/eventthread.h

```diff
@@ -37,7 +37,7 @@
 #include <stdint.h>
 
 struct RGSSThreadData;
-typedef struct ALCdevice_struct ALCdevice;
+typedef struct ALCdevice ALCdevice;
```

Comment:
This should also work with both versions: `struct ALCdevice;`
Can you try this?

Reply:
Yeah, I can try this.

Reply:
Sadly doesn't work. We can either do preprocessor #if or include alc.h in the header.
https://github.com/kcat/openal-soft/commit/399dfca1e3352050a5289be6bcd7b7dce353ccf8
openal-soft doesn't seem to consider using the struct as public api, so including alc.h seems to be only sane way. Sadly they typedef their thing too so we can't really forward declare. (I'm not really fond of typedef)